examples/metadata/components/lockstore.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: lockstore
```

Comment:
why is this component needed in metadata example?

Reply:
Not really needed. I thought the more the merrier but I guess the 2 "traditional" components (pubsub and state store on redis)  should be more than enough. I will remove this one.

Reply:
Any component that works with the standard Dapr installation is fine. I don't care which one we are using @mukundansundar